### PR TITLE
Update: Wider thumbnails

### PR DESCRIPTION
**Pull request by ConradJChan**
Increases the thumbnails sidebar width by 25px. Also decreases the selected state border to 3px

Before:
Screen Shot 2019-03-25 at 3 08 32 PM

Now:
Screen Shot 2019-03-25 at 3 07 52 PM

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @ConradJChan has signed the CLA. Thanks for the pull request!

File: test/integration/document/Thumbnails.e2e.test.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ describe('Preview Document Thumbnails', () => {
      * @return {Assertion} Chai Assertion
      */
     const notBeVisible = ($thumbnailsSidebar) =>
-        expect($thumbnailsSidebar).to.have.css('transform', 'matrix(1, 0, 0, 1, -201, 0)'); // translateX(-201px)
+        expect($thumbnailsSidebar).to.have.css('transform', 'matrix(1, 0, 0, 1, -226, 0)'); // translateX(-226px)
```
Comment (jeremypress) [inline test/integration/document/Thumbnails.e2e.test.js]: Should we just check for transform? Then we don't need to worry about a change in value.

File: src/lib/viewers/doc/_docBase.scss [hunk 2]
```diff
@@ -49,13 +49,12 @@ $thumbnail-sidebar-width: 201px;
 
     .bp-thumbnail-nav {
         background-color: $d-eight;
-        border: $thumbnail-border-radius solid $ffive;
+        border: 3px solid $ffive;
```
Comment (jeremypress) [inline src/lib/viewers/doc/_docBase.scss]: Update the variable and use it?

File: test/integration/document/Thumbnails.e2e.test.js (hunk 1 above)
Comment (ConradJChan) [reply test/integration/document/Thumbnails.e2e.test.js]: What I want to verify is that the transform value changed from 0 to 226 or vice versa, to reflect the state of the sidebar toggling

File: src/lib/viewers/doc/_docBase.scss [hunk 3]
```diff
@@ -55,7 +55,6 @@ $thumbnail-sidebar-width: 201px;
         flex: 1 0 auto;
         overflow: hidden;
         padding: 0;
-        width: 134px;
```
Comment (jstoffan) [inline src/lib/viewers/doc/_docBase.scss]: Was this not needed?

Comment (ConradJChan) [reply src/lib/viewers/doc/_docBase.scss]: Yeah I don't think so because the flexbox width seems to take precedence

Comment (ConradJChan) [reply src/lib/viewers/doc/_docBase.scss]: I take it back, ie11 needs this, but I'll set it to the flex-basis instead

**Pull request merged**